### ISSUE TITLE
Content is not loading

### ISSUE THREAD

**Issue by Bantolomeus** [Bantolomeus/bookmaster4000mkii#115]
As seen in the screenshot, the backend content does not load and at some point it simply does not continue to call for it, so I have to reload the page multiple times, like 6-7 times.
![bugLoading](https://user-images.githubusercontent.com/1318613/146653571-fb798427-9d04-4b8a-b514-712144678d06.PNG)

**Comment by Husterknupp** [comment]
<img width="506" alt="image" src="https://user-images.githubusercontent.com/1265382/146775672-77b94eb6-96fc-4706-83c2-c09ecf6a3b69.png">

@Bantolomeus the latest `master` jar has Spring Boot's actuator feature turned on (I added a pom dependency). Start the jar as usual and open the `http://localhost:8080/actuator/httptrace` endpoint. You will find the recent 100 http requests/responses and a `timeTaken` property (see [the Spring docs](https://docs.spring.io/spring-boot/docs/current/actuator-api/htmlsingle/#http-trace.retrieving.response-structure)).

I'd expect that for failing requests, also the timeTaken in the actuator endpoint should be high. Can you verify that?

**Comment by Husterknupp** [comment]
@Bantolomeus the screenshot in the description is not super helpful. The error in the browser console seems to suggest that you hit refresh so fast that new requests aborted old requests (hence the `"xhrStatus":"abort"`). But these are requests that kind of are no real requests - it doesn't show the failing or slow requests.

Can you verify that and see if you have problems with requesting the REST controllers also when using Postman or cURL or so (instead of the Angular UI)

---

To sum it up, two things:
1. Reproduce bad behavior with another tool
2. Double check the long process time witht the actuator endpoint

**Comment by Bantolomeus** [comment]
K, will do it if it is slow again. Thats kinda the problem, it is not always slow, but we will see.

**Comment by Husterknupp** [comment]
@Bantolomeus did you find this happening again, could you gather some debug info?

**Comment by Bantolomeus** [comment]
It did not happened again. I think the problem could have overlapped with Tarkov, which puts my system at its limits. So we probably have no issue. I will close this issue and create a new issue if it comes up again and I have data to analyse.
